test/WebJobs.Script.Tests/Workers/Rpc/GrpcWorkerChannelTests.cs

```diff
@@ -487,6 +487,21 @@ public void ReceivesInboundEvent_FunctionLoadResponse()
             Assert.True(traces.Any(m => string.Equals(m.FormattedMessage, "Received FunctionLoadResponse for function: 'js1' with functionId: 'TestFunctionId1'.")));
         }
 
+        [Fact]
+        public void ReceivesInboundEvent_FunctionLoadResponses()
```

Comment:
Can you think of any failure scenarios that we haven't tested at all? I'd like us to test more of what happens when things go wrong. Also is it possible to add an integration test for this? Maybe something under: test/WebJobs.Script.Tests.Integration/Rpc